README.md

```diff
@@ -41,6 +41,24 @@ MineColonies crashes every time? Have a suggestion? Found a bug? Create an issue
 
 ## For Developers
 
+### Adding MineColonies as Maven Dependecy

```

Comment:
Maven?

Reply:
As is "link" to the maven repo. But I could change it to something else if you want to.

Reply:
I meant more as in, you are saying "Maven" dependency, but you are showing Gradle code.

Also "Dependency" is misspelled I just noticed.

Reply:
There is no other kind of dependency
So please remove the "Maven" from the statement.